relational_operators/CreateTableOperator.cpp

```diff
@@ -20,27 +20,15 @@
 #include <memory>
 
 #include "catalog/CatalogDatabase.hpp"
-#include "query_execution/WorkOrdersContainer.hpp"
-
-#include "glog/logging.h"
 
 namespace quickstep {
 
 bool CreateTableOperator::getAllWorkOrders(WorkOrdersContainer *container) {
   if (!work_generated_) {
     work_generated_ = true;
-    container->addNormalWorkOrder(
-        new CreateTableWorkOrder(relation_.release()),
-        op_index_);
+    database_->addRelation(relation_.release());
```

Comment:
This is a somewhat hacky abuse of `getAllWorkOrders()`, in that it does actual work in a method that is only supposed to *produce* work.

I do agree that there is a need for "foreman only" operators, but we should probably make that explicit rather than hiding it here.